include/seqan3/alphabet/quality/phred94.hpp

```diff
@@ -0,0 +1,119 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Lydia Buntrock <lydia.buntrock AT fu-berlin.de>
+ * \brief Provides seqan3::phred94 quality scores.
+ */
+
+#pragma once
+
+#include <seqan3/alphabet/quality/quality_base.hpp>
+
+// ------------------------------------------------------------------
+// phred94
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+
+/*!\brief Quality type for PacBio Phred scores of HiFi reads.
+ * \implements seqan3::writable_quality_alphabet
+ * \if DEV \implements seqan3::detail::writable_constexpr_alphabet \endif
+ * \implements seqan3::trivially_copyable
+ * \implements seqan3::standard_layout
+ * \implements std::regular
+ *
+ * \ingroup quality
+ *
+ * \details
+ *
+ * The phred94 quality alphabet represents the zero-based phred score range [0..93] mapped to the ASCII range
+ * ['!' .. '~'] (Sanger, Illumina 1.8+ format). It represents the PacBio use case, typically HiFi reads. For Sanger and
+ * Illumina phred scores of raw reads the range is (0 to 41), represented as seqan3::phred42, and the larger score type
+ * (0 to 62), represented as seqan3::phred63.
```

Comment:
```suggestion
 * Illumina phred scores of raw reads the range is typically (0 to 41), represented as seqan3::phred42. If you expect only slightly  
 * larger score types you can use seqan3::phred63 (0 to 62) which still has memory advantages when used with `seqan3::qualified`.
```

check line length after applying the suggestion.